Final Project - Joe Bartone, Chris Mercer, and Danny Sullivan

Mersair opened the pull request.

Commit:
Finalized proposal

Mersair committed:
Update README.md

Mersair committed:
Rename README.md to proposal.md

charlieroberts commented:
This sounds like a great idea. I really like the concept of sharing freely available online resources as well... I'd imagine this could extend to useful web-based tutorials / videos / examples in addition to online textbooks. Could be a nice one-stop shop for resources that students have found useful in classes at WPI!

Commit:
All pages serving

Commit:
User logic

Commit:
🌆🌓 Checkpoint
./package.json:6267243/2661
./app.js:6267243/2

Commit:
👼👬 Checkpoint
./public/javascripts/client.js:6267243/792
./public/pages/home.html:6267243/29
./routes/apiUser.js:6267243/1188
./public/javascripts/login.js:6267243/757

Commit:
📺💡 Checkpoint
./public/javascripts/login.js:6267243/1917
./public/pages/home.html:6267243/29

Commit:
🛍🎇 Checkpoint
./public/pages/home.html:6267243/16
./routes/apiUser.js:6267243/10
./public/pages/index.html:6267243/4413
./public/javascripts/login.js:6267243/48
./app.js:6267243/4

Commit:
🏚🚿 Checkpoint
./app.js:6267243/2
./routes/home.js:6267243/715
./routes/apiUser.js:6267243/70
./public/javascripts/login.js:6267243/33
./public/javascripts/create.js:6267243/1002

Commit:
🚏👩 Checkpoint
./routes/apiUser.js:6267243/114
./public/javascripts/login.js:6267243/320

Commit:
😎🏵 Checkpoint
./public/javascripts/login.js:6267243/619
./routes/apiUser.js:6267243/177

Commit:
changed a lil login logic

Commit:
🙍🎀 Checkpoint
./public/javascripts/login.js:6267243/1749
./public/javascripts/create.js:6267243/305
./routes/apiUser.js:6267243/142
./public/javascripts/client.js:6267243/401
./public/pages/index.html:6267243/583

Commit:
🛰🏂 Checkpoint
./public/pages/index.html:6267243/439
./public/pages/create.html:6267243/5713
./public/pages/messaging.html:6267243/3306
./public/pages/search.html:6267243/4594
./public/pages/add.html:6267243/4010
./routes/add.js:6267243/282
./app.js:6267243/144

Commit:
add book page

Commit:
lil change

Commit:
👫🐧 Checkpoint
./routes/apiUser.js:6266233/3820

Commit:
🔮👕 Checkpoint
./routes/apiUser.js:6266233/781

Commit:
🐫👵 Checkpoint
./routes/apiUser.js:6266233/727

Commit:
🐺🏪 Checkpoint
./routes/apiUser.js:6266233/932

Commit:
🐑💇 Checkpoint
./routes/apiUser.js:6266233/2898

Commit:
💪🏠 Checkpoint
./routes/apiUser.js:6266233/2148

Commit:
⛴🌖 Checkpoint
./routes/apiUser.js:6266233/266

Commit:
😼⛹ Checkpoint
./routes/apiUser.js:6266233/1680

Commit:
❣️🌧 Checkpoint
./routes/apiUser.js:6266233/1260

Commit:
👂🕵 Checkpoint
./routes/apiUser.js:6266233/1847

Commit:
🌚⛑ Checkpoint
./routes/apiUser.js:6266233/1680

Commit:
🎫🙏 Checkpoint
./routes/apiUser.js:6266233/9100

Commit:
🏠🎐 Checkpoint
./routes/apiUser.js:6266233/723
./public/javascripts/create.js:6266233/83
./public/pages/create.html:6266233/88

Commit:
💓🌼 Checkpoint
./routes/apiUser.js:6266233/288

Commit:
messaging

Commit:
merge

Commit:
lil baby change

Commit:
🎺🐒 Checkpoint
./public/pages/messaging.html:6267243/1066

Commit:
working on msg

DannyJSullivan committed:
get all users plus some other user cleanup

Commit:
🍀📠 Checkpoint
./public/javascripts/messaging.js:6267243/6584

Commit:
🎽🍭 Checkpoint
./public/javascripts/messaging.js:6267243/2650

DannyJSullivan committed:
base queries for books

Commit:
🌞🍟 Checkpoint
./public/javascripts/messaging.js:6267243/453

Commit:
merge?

Commit:
🍂🌸 Checkpoint
./public/javascripts/client.js:6267243/14269

Commit:
lil client

Commit:
🍄🗻 Checkpoint
./public/javascripts/client.js:6267243/8446
./routes/apiBook.js:6267243/20178

Commit:
yo

Commit:
📮🚔 Checkpoint
./routes/apiBook.js:6267243/11108
./public/javascripts/client.js:6267243/11557
./app.js:6267243/4

Commit:
🚶🎯 Checkpoint
./public/javascripts/client.js:6267243/16345
./app.js:6267243/2
./public/javascripts/create.js:6267243/2
./routes/apiBook.js:6267243/70

Commit:
😄🎓 Checkpoint
./public/javascripts/client.js:6267243/176
./public/pages/add.html:6267243/515
./public/pages/search.html:6267243/14

Commit:
🎩⚾️ Checkpoint
./public/javascripts/client.js:6267243/60
./public/javascripts/search.js:6267243/7030
./public/pages/search.html:6267243/27

Commit:
🚵🐜 Checkpoint
./public/javascripts/search.js:6267243/8

Commit:
🎢🎈 Checkpoint
./public/javascripts/search.js:6267243/2899

Commit:
🐫😘 Checkpoint
./public/javascripts/search.js:6267243/4291

Commit:
🕴🙈 Checkpoint
./public/javascripts/search.js:6267243/1704
./public/pages/search.html:6267243/516

Commit:
🏑🌟 Checkpoint
./public/javascripts/search.js:6267243/20

DannyJSullivan committed:
plz close db sessions

DannyJSullivan committed:
Merge branch 'master' of https://github.com/Mersair/fp

Commit:
bruh

Commit:
merge

Commit:
🎤🙋 Checkpoint
./public/javascripts/search.js:6267243/4443

Commit:
🎂🚇 Checkpoint
./public/pages/search.html:6267243/125
./public/javascripts/search.js:6267243/637
./public/pages/about.html:6267243/1402

DannyJSullivan committed:
its your boy location functionality

Commit:
💧🚐 Checkpoint
./public/pages/messaging.html:6267243/7516

DannyJSullivan committed:
Merge remote-tracking branch 'origin/master'

Commit:
Login on enter

Commit:
Merge branch 'master' of github.com:Mersair/fp

Commit:
👂🏥 Checkpoint
./public/pages/messaging.html:6267243/597
./public/javascripts/messaging.js:6267243/1084

Commit:
Merge branch 'master' of https://github.com/Mersair/fp

Commit:
💳🎈 Checkpoint
./public/javascripts/messaging.js:6267243/848
./public/pages/messaging.html:6267243/14

Commit:
Logout wipes cookie

Commit:
Merge branch 'master' of github.com:Mersair/fp

DannyJSullivan committed:
messaging plz work

DannyJSullivan committed:
Merge branch 'master' of https://github.com/Mersair/fp

Commit:
prepull

Commit:
merge

Commit:
Rework account creation flow

Commit:
Merge branch 'master' of github.com:Mersair/fp

Commit:
👓🌟 Checkpoint
./public/pages/messaging.html:6267243/1063

Commit:
Bypass login screen if user already had cookie

DannyJSullivan committed:
we hate react

DannyJSullivan committed:
Merge branch 'master' of https://github.com/Mersair/fp

Commit:
📊🕹 Checkpoint
./public/pages/messaging.html:6267243/153
./public/javascripts/messaging.js:6267243/144

Commit:
🍬🎃 Checkpoint
./public/pages/messaging.html:6267243/10
./public/javascripts/messaging.js:6267243/70
./public/stylesheets/style.css:6267243/128

Commit:
🐃🚡 Checkpoint
./public/stylesheets/style.css:6267243/1
./public/javascripts/messaging.js:6267243/5366

Commit:
✌️🙌 Checkpoint
./public/javascripts/messaging.js:6267243/10638

Commit:
Add links to digital content

Commit:
Merge branch 'master' of github.com:Mersair/fp

Commit:
messsssss

Commit:
marge

Commit:
Rename Physical

Commit:
Merge branch 'master' of github.com:Mersair/fp

Commit:
I can't read -- changed Physical to Digital

Commit:
⚽️🌨 Checkpoint
./public/javascripts/messaging.js:6267243/934

DannyJSullivan committed:
we can now add books

DannyJSullivan committed:
Merge branch 'master' of https://github.com/Mersair/fp

Commit:
Catch empty location entries

Commit:
Merge branch 'master' of github.com:Mersair/fp

Commit:
margarine

Commit:
📅👁 Checkpoint
./public/javascripts/messaging.js:6267243/443
./public/pages/messaging.html:6267243/5825

DannyJSullivan committed:
edit book db side plz

Commit:
🙉🗻 Checkpoint
./public/javascripts/messaging.js:6267243/8293
./public/pages/messaging.html:6267243/161

Commit:
meeeeeeerge

Commit:
🙉📈 Checkpoint
./public/javascripts/messaging.js:6267243/550
./public/pages/messaging.html:6267243/72

Commit:
🍏🏀 Checkpoint
./public/javascripts/messaging.js:6267243/355
./public/pages/messaging.html:6267243/53

Commit:
messaging is hot hot hot

Commit:
🚔⛷ Checkpoint
./public/javascripts/messaging.js:6267243/8696
./public/pages/messaging.html:6267243/94

DannyJSullivan committed:
emergency db clears if necessary

DannyJSullivan committed:
Merge branch 'master' of https://github.com/Mersair/fp

Commit:
merge

DannyJSullivan committed:
typo

DannyJSullivan committed:
Merge branch 'master' of https://github.com/Mersair/fp

Commit:
🍜🏌 Checkpoint
./public/javascripts/messaging.js:6267243/333

Commit:
smol

Commit:
🚌🍼 Checkpoint
./public/javascripts/messaging.js:6267243/8420
./public/pages/messaging.html:6267243/25

Commit:
Allow editing

Commit:
merge

Commit:
👨‍👨‍👦⛩ Checkpoint
./public/pages/messaging.html:6267243/70

Commit:
Delete books

Commit:
Delete books

Commit:
Fixed form field names for deleting book

Commit:
💶🏫 Checkpoint
./public/pages/messaging.html:6267243/115
./public/javascripts/messaging.js:6267243/1757

Commit:
Redirect for delete

Commit:
maaaaarge

Commit:
👖🍣 Checkpoint
./public/javascripts/messaging.js:6267243/267
./public/pages/messaging.html:6267243/8

Commit:
search location finder

DannyJSullivan committed:
😊🔫

DannyJSullivan committed:
😊🔫

Commit:
⚽️🐴 Checkpoint
./public/javascripts/messaging.js:6267243/867

Commit:
final merge

Commit:
💃🛀 Checkpoint
./public/javascripts/messaging.js:6267243/442
./public/pages/messaging.html:6267243/32

Commit:
README

Commit:
merge

Commit:
done

Commit:
🍃🍎 Checkpoint
./public/javascripts/create.js:6267243/211
./public/javascripts/messaging.js:6267243/405

Commit:
👹🚨 Checkpoint
./public/javascripts/search.js:6267243/313
./public/javascripts/add.js:6267243/575
./public/javascripts/messaging.js:6267243/2
